src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+  static deleteIndex = Animal.alive.findIndex((animal) => animal.health <= 0);
+
+  deleteAnimal() {
```

Comment:
The `deleteAnimal` method should be static and should not be called on an instance of `Animal`. It should be responsible for removing dead animals from the `Animal.alive` array.